README.md

```diff
@@ -1555,6 +1555,8 @@ IEEE Robotics and Automation Letters, 7(3):8146-8153, 2022. [PDF](https://www.ir
 - <a name="Ji23arxiv"></a>Ji, X., Wei, J., Wang, Y., Shang, H., Kneip, L.,  
 *[Cross-modal Place Recognition in Image Databases using Event-based Sensors](https://arxiv.org/abs/2307.01047)*,  
 arXiv, 2023.
+*[How Many Events Do You Need? Event-Based Visual Place Recognition Using Sparse But Varying Pixels](https://ieeexplore.ieee.org/abstract/document/9925670)*,  
```

Comment:
Using the DOI is preferable: https://doi.org/10.1109/LRA.2022.3216226

Reply:
Please look at the preview:
![image](https://github.com/user-attachments/assets/035c3624-2e85-4b25-9bfd-381444ad6d9b)

Authors are missing, and something is wrong with the formatting. This entry is mixing with the previous entry in the list ("Cross-modal place recognition"...). Please fix it.

Reply:
Sorry about that, I'll take a quick look.

Reply:
Hopefully it's fixed now.